### PR TITLE
IGAPP-1201: Fix sticky footer in Event List

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: release-notes/unreleased/IGAPP-1201-fix-sticky-footer.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1201
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - android
+  - ios
+en: Fixed footer not sticking to the bottom in the event list.
```
Comment (steffenkleinle) [inline release-notes/unreleased/IGAPP-1201-fix-sticky-footer.yaml:6]:
```suggestion
issue_key: IGAPP-1201
show_in_stores: true
platforms:
  - android
  - ios
en: Fixed footer not sticking to the bottom in the event list.
de: Die Fußzeile ist in der Eventliste jetzt wieder unten fixiert.
```

**Review by steffenkleinle — COMMENTED** [review]
Works if no events are available but not if there is a list of events (at least for me).

**Comment by sarahsporck** [comment]
> Works if no events are available but not if there is a list of events (at least for me).

Will look at it again.

**Review by f1sh1918 — APPROVED** [review]
not tested

**Review by steffenkleinle — APPROVED** [review]
Tested on android, nice!

**Pull request merged**